multinet/api.py

```diff
@@ -25,11 +27,62 @@
     AlreadyExists,
     RequiredParamsMissing,
 )
-from multinet.user import current_user
+from multinet.user import current_user, find_user_from_id
 
 bp = Blueprint("multinet", __name__)
 
 
+# Included here due to circular imports
+# TODO: Remove once implementing new ORM and permission storage
```

Comment:
Is your plan to finish the ORM work first, then rebase on (or merge from) that work here?

Reply:
I figured that this would make it in first, so the comment was there to leave a breadcrumb for circling back. Do you think it's worth waiting for the ORM work to make it in first? Managing permissions from the client relies on this endpoint.

Reply:
If the circular dependency is causing test failures, I'm not sure that we should simply merge it in. If there's an easy way to fix that issue, then we should do it. If that will cause a major headache then we may need to go out on a limb and merge in "broken" code but I'd like to avoid that if possible.

What do you think?

Reply:
The main reason I didn't solve the circular dependency issue in a more robust way is because it seemed a bit out of scope for this issue, and the functions that are causing them to happen (these conversion functions) are temporary and would be removed when the ORM change comes down. Also, if we encounter this same issue during the ORM refactor, that's a perfectly valid time to fix it. I just didn't want to spend a ton of time reorganizing the code base just for it not to matter once we ultimately reorganize it for the ORM.

Reply:
I agree with what you're saying. But this will not work if the server can't actually run because of the circular import. If you can certify that this does work properly, then we're all good. I would just ask you to file an issue pointing to these code blobs (a TODO in the codebase isn't quite as trackable as I'd like).

Reply:
> I agree with what you're saying. But this will not work if the server can't actually run because of the circular import. If you can certify that this does work properly, then we're all good.

This does work properly (as in, no circular imports at runtime), although having someone else test this out may be a good idea.

> I would just ask you to file an issue pointing to these code blobs (a TODO in the codebase isn't quite as trackable as I'd like).

#442